tools/tuf/test/build_and_push_orbit.sh

```
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -x
+
+source ./tools/tuf/test/load_orbit_version_vars.sh
```

Shellcheck (0.11.0)

[info] Not following: ./tools/tuf/test/load_orbit_version_vars.sh was not specified as input (see shellcheck -x).

(SC1091)